tensorflow/lite/core/c/builtin_op_data.h

```diff
@@ -654,6 +655,13 @@ typedef struct {
   size_t attributes_size;
 } TfLiteStablehloCompositeParams;
 
+typedef struct {
+  // See the stablehlo spec for the explanation of the attributes:
+  // https://github.com/openxla/stablehlo/blob/main/docs/spec.md#case
+  int32_t branch_subgraph_indices[TFLITE_STABLEHLO_CASE_PARAMS_MAX_BRANCHES_COUNT];
+  int num_branches;
```

Comment:
`num_branches` should be an unsigned type

Reply:
okay